### PR TITLE
poac 0.7.0

### PR DESCRIPTION
**Pull request by ken-matsui**
Created with `brew bump-formula-pr`.<details>
  <summary>release notes</summary>
  <pre>## What's Changed
* Try macos-13 runner by @ken-matsui in https://github.com/poac-dev/poac/pull/762
* Fix ninja manifest file name from `ninja.build` to `build.ninja` by @ken-matsui in https://github.com/poac-dev/poac/pull/764
* Include project headers by @ken-matsui in https://github.com/poac-dev/poac/pull/766
* Output `compile_commands.json` by @ken-matsui in https://github.com/poac-dev/poac/pull/767
* Organize cache directories for git support by @ken-matsui in https://github.com/poac-dev/poac/pull/775
* Migrate sources to C++20 modules by @ken-matsui in https://github.com/poac-dev/poac/pull/776
* Drop fmt 8 support by @ken-matsui in https://github.com/poac-dev/poac/pull/783
* Simplify core.builder.compiler.cxx by @ken-matsui in https://github.com/poac-dev/poac/pull/784
* Rename mod.cmake -> Mod.cmake by @ken-matsui in https://github.com/poac-dev/poac/pull/785
* Delete termcolor2 config header by @ken-matsui in https://github.com/poac-dev/poac/pull/786
* Bump actions/upload-pages-artifact from 1 to 2 by @dependabot in https://github.com/poac-dev/poac/pull/791
* Bump actions/checkout from 3 to 4 by @dependabot in https://github.com/poac-dev/poac/pull/797
* Format codes by @ken-matsui in https://github.com/poac-dev/poac/pull/798
* Full reimplementation by @ken-matsui in https://github.com/poac-dev/poac/pull/799
* Support compile_commands.json by @ken-matsui in https://github.com/poac-dev/poac/pull/800
* Support --exclude for lint command by @ken-matsui in https://github.com/poac-dev/poac/pull/801

**Full Changelog**: https://github.com/poac-dev/poac/compare/0.6.0...0.7.0</pre>
</details>

### CODE REVIEW

File: Formula/p/poac.rb [hunk 1]
```diff
@@ -19,15 +18,6 @@ class Poac < Formula
...
   end
 
-  depends_on "cmake" => :build
-  depends_on "boost"
-  depends_on "fmt"
-  depends_on "libgit2"
-  depends_on "openssl@3"
-  depends_on "spdlog"
-
-  uses_from_macos "libarchive"
```
Comment (chenrui333) [inline Formula/p/poac.rb:29]: any PR ref for such change? (seems like a big dependency change)

Comment (ken-matsui) [reply Formula/p/poac.rb:29]: @chenrui333 Yes, here it is: https://github.com/poac-dev/poac/pull/799

Comment (chenrui333) [reply Formula/p/poac.rb:29]: Thanks!

File: Formula/p/poac.rb [hunk 2]
```diff
@@ -41,23 +31,13 @@ class Poac < Formula
...
-    inreplace "cmake/AddOpenSSL.cmake", "${POAC_HOMEBREW_ROOT_PATH}/openssl",
-                                        Formula["openssl@3"].opt_prefix
-
-    system "cmake", "-S", ".", "-B", "build", "-DPOAC_BUILD_TESTING=OFF", *std_cmake_args
-    system "cmake", "--build", "build"
-    system "cmake", "--install", "build"
-
-    man.install "src/etc/man/man1"
-    bash_completion.install "src/etc/poac.bash" => "poac"
-    zsh_completion.install_symlink bash_completion/"poac" => "_poac"
```
Comment (chenrui333) [inline Formula/p/poac.rb:55]: Are these supported in the new build?

Comment (ken-matsui) [reply Formula/p/poac.rb:55]: @chenrui333 Unfortunately, these are removed to reduce maintenance costs. That said, I am willing to support these in future releases.

Comment (chenrui333) [reply Formula/p/poac.rb:55]: got you, thanks!

**Review by chenrui333 — APPROVED** [review]

**Pull request merged**